NeoMathEngine/src/GPU/CUDA/CudaMathEngine.cu

```diff
@@ -49,13 +49,15 @@ CCudaMathEngine::CCudaMathEngine( const CCusparse* _cusparse, const CCublas* _cu
 {
 	device.swap( _device );
 
+	CDllLoader::Load( CDllLoader::CUDA_DLL );
```

Comment:
Why is it here?

Reply:
Correct order:
(constructor)
- load libs
- initialize cublas
- initialize cusparse
- intialize allocator
(destructor)
- ... reverse order of constructor ...